src/Components/Publishing/Display/DisplayPanel.tsx

```diff
@@ -234,6 +277,7 @@ const DisplayPanelContainer = Div`
     margin: auto;
   `}
 `
+DisplayPanelContainer.displayName = "DisplayPanelContainer"
```

Comment:
Seeing a duplicate of this line below --

Reply:
This was fixed in https://github.com/artsy/reaction/pull/361